src/main/java/List.java

```diff
@@ -0,0 +1,18 @@
+public class List {
```

Comment:
Good that you have a List class to print the tasks instead of using a loop in main function